src/components/List/SortBySwitches.tsx

```diff
@@ -0,0 +1,103 @@
+import { List } from "@/pages/list";
+import { RadioGroup } from "@headlessui/react";
+import { Dispatch, SetStateAction } from "react";
+import { clsx } from "clsx";
+
+export function sortByAlphabet(input: List) {
+  const sorted = [...input];
+  sorted.sort((a, b) => {
+    const nameA = a.item.toUpperCase();
+    const nameB = b.item.toUpperCase();
+    if (nameA < nameB) {
+      return -1;
+    }
+    if (nameA > nameB) {
+      return 1;
+    }
+    return 0;
+  });
+  return sorted;
+}
+export function sortByCategory(input: List): List {
+  const sorted = [...input];
+  sorted.sort((a, b) => {
+    const categoryA = a.category.toUpperCase();
+    const categoryB = b.category.toUpperCase();
+
+    const nameA = a.item.toUpperCase();
+    const nameB = b.item.toUpperCase();
+
+    if (categoryA < categoryB) {
+      return -1;
+    }
+    if (categoryA > categoryB) {
+      return 1;
+    }
+
+    if (categoryA === categoryB) {
+      if (nameA < nameB) {
+        return -1;
+      }
+      if (nameA > nameB) {
+        return 1;
+      }
+      return 0;
+    }
+    return 0;
+  });
+  return sorted;
+}
+
+export function sortByDate() {}
```

Comment:
Is this intentionally empty?

Reply:
Its emtpy cause there we didnt decide yet if and how to implement date and time, shall i implement the function in this branch or rather have a new issue created for it and doing it in a new branch after merge ?